charts/latest/blob-csi-driver/templates/csi-blob-node.yaml

```diff
@@ -13,6 +13,10 @@ spec:
 {{ include "blob.labels" . | indent 6 }}
         app: csi-blob-node
     spec:
+      {{- if .Values.imagePullSecrets }}
```

Comment:
only need to make change under charts/latest, and then run `helm package charts/latest/blob-csi-driver -d charts/latest/` to update tgz file, thanks.

Reply:
Done(?)